travis-ymls/commons-beanutils.travis.yml

```diff
@@ -0,0 +1,51 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : commons-beanutils
+# Source Repo         : https://github.com/apache/commons-beanutils.git
+# Travis Job Link     : https://travis-ci.com/github/genisysram/commons-beanutils/jobs/484481140
+# Created travis.yml  : No
+# Maintainer          : <ramakrishna.s@genisys-group.com>
+#                       <sethp@us.ibm.com>
+#
+# Script License      : Apache License, Version 2 or later
+#
+# ----------------------------------------------------------------------------
+
+
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+language: java
+sudo: false
+
+arch: 
+    - amd64
+    - ppc64le
+
+jdk:
+  - oraclejdk11
+  - openjdk8
+  - openjdk11
+  - openjdk15
+  - openjdk-ea
+
+before_install:
+        - mkdir -p /opt/maven
```

Comment:
Can we add as to why this manual installation of mvn required ? Also is it required for intel also ?